godot-ffi/src/lib.rs

```diff
@@ -148,6 +149,32 @@ pub fn default_call_error() -> GDExtensionCallError {
     }
 }
 
+#[doc(hidden)]
+#[inline]
+pub fn panic_on_call_error(err: &GDExtensionCallError) {
```

Comment:
a little late, but this seems like a great candidate for using `#[call_tracker]`, doesn't it?

Reply:
This is great advice, thanks! I was actually considering to make it a macro for this very reason, but totally forgot about this attribute 👍 

No worries about "late", 2nd round of test improvements already on the way 😎

Reply:
I just tested it btw, indeed it works and shows the occurrence of the panic (with `RUST_BACKTRACE=0`) on the call site.